src/main/scala/net/degoes/03-nulls.scala

```diff
@@ -75,34 +79,42 @@ object Nulls extends ZIOSpecDefault {
           test("map") {
             val option: Option[Int] = Some(42)
 
-            def convert(o: Option[Int]): Option[Char] = ???
+            def convert(o: Option[Int]): Option[Char] = o match {
```

Comment:
You can do `o.map` directly, it works even if o is None (in that case it will still be None)